doc/manual/config-basic.rst

```diff
@@ -40,6 +40,16 @@ A jury or administrative user can also be associated with a team. This
 will enable that user to submit solutions to the system, or resubmit
 edited team solutions.
 
+Resetting the password for a user
+---------------------------------
+
+If you do not have access anymore to any admin user, you can use the following
+command to reset the password of a user to a random value::
```

Comment:
nit: make it clear that the new password will be displayed and is not a script argument